home.html

```diff
@@ -0,0 +1,330 @@
+        </div>
+        <div class="row">
+          <div class="col-4 band-image hide-mobile">
+            <img src="./images/bands/nickelback.jpeg" alt="image of the members of Green Day" width="150px">
+          </div>
+          <div class="col-8 hide-mobile">
+            <h3>Nickelback</h3>
+            <p class="band-origin">A Canadian rock band formed in Hanna, Alberta, Canada in 1995.</p>
+            <div class="separator"></div>
+            <p class="band-highlights">Won the Billboard Music Award for Top Duo/Group in 2006 and the Juno Award for Album of the year in 2009; among others.</p>
+          </div>
+          <hr class="hide-mobile">
+        </div>
+        <div class="row">
+          <div class="col-4 band-image hide-mobile">
+            <img src="./images/bands/red_hot_chili_peppers.jpeg" alt="image of the members of Green Day" width="150px">
+          </div>
+          <div class="col-8 hide-mobile">
+            <h3>Red Hot Chili Peppers</h3>
+            <p class="band-origin">An American rock band formed in Los Angeles, California in 1982.</p>
+            <div class="separator"></div>
+            <p class="band-highlights">Grammy Award winner for Best Rock Album and Best Rock Performance in 2007; along with many other awards and nominations.</p>
+          </div>
+          <hr class="hide-mobile">
+        </div>
+      </div>
+      <div class="main-button-mobile more-button">
+        <button class="main-more-button">MORE<i class="fa-solid fa-caret-down"></i></button>
+      </div>
+      <div class="main-button-mobile less-button hide-mobile">
+        <button class="main-less-button hide-mobile">LESS<i class="fa-solid fa-caret-up"></i></button>
+      </div>
+    </div>
+  </section>
+  <section class="partners-container">
+    <div class="partner-title">
+      <h2>Partner</h2>
+    </div>
+    <div class="title-underline"></div>
+    <div class="partner-logo-container">
+      <div class="partners">
+        <!-- <div> -->
+          <img src="./images/partner_logos/mozilla.png" alt="">
+        <!-- </div> -->
+        <div>
+          <img src="./images/partner_logos/google.png" alt="">
+        </div>
+        <div>
+          <img src="./images/partner_logos/naver.png" alt="">
+        </div>
+        <div>
+          <img src="./images/partner_logos/daumkakao.png" alt="">
+        </div>
+        <div>
+          <img src="./images/partner_logos/airbnb.png" alt="">
+        </div>
+      </div>
+    </div>
+  </section>
+  <footer>
+    <div class="container">
+      <div class="row row-cols-md-2">
+        <div class="col-6 d-flex justify-content-center cc-logo-container">
+          <img class="positive" src="./images/footer/positive.png" alt="creative commons korea logo in positive">
+        </div>
+        <div class="col-6 d-flex justify-content-center cc-logo-container">
+            <p>2023 Creative Commons korea.<br>Some Rights Reserved.</p>
+        </div>
```

Comment:
- [ ] Kindly fix the space between the image and the copyright.

<img width="1440" alt="Screenshot 2023-02-17 at 15 06 53" src="https://user-images.githubusercontent.com/81444338/219814111-91b0a1a2-11bb-4457-9afa-042d0759b352.png">